ansible/roles/cri/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+# Instructions: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker
```

Comment:
This looks good, although we've removed the originally docker option.

We should try to use the example of how the CNI was done, and use the same method for the CRI

See an example here: https://github.com/raspbernetes/k8s-cluster-installation/tree/master/ansible/roles/cni/tasks